lib/displayHelpers.ts

```diff
@@ -49,12 +43,21 @@ const printableCoin = (coin: Coin, chainInfo: ChainInfo) => {
   }
 
   // Auto-convert leading "u"s
-  if (coin.denom?.startsWith("u")) {
+  if (coin.denom.startsWith("u")) {
     const value = Decimal.fromAtomics(coin.amount ?? "0", 6).toString();
     const ticker = coin.denom.slice(1).toUpperCase();
     return value + thinSpace + ticker;
   }
 
+  // Ellide IBC tokens
+  if (coin.denom.startsWith("ibc/")) {
+    const value = Decimal.fromAtomics(coin.amount ?? "0", 6).toString();
+    const hash = coin.denom.slice(4);
+    const ellidedHash = ellideMiddle(hash, 12);
```

Comment:
```suggestion
    const ellidedHash = ellideMiddle(hash, 11);
```

This makes it 5 chars on each side which is enough